tests/unit/inference/test_inference.py

```diff
@@ -323,6 +334,66 @@ def test(
         assert assert_fn(bs_output, ds_output)
 
 
+@pytest.mark.seq_inference
+@pytest.mark.parametrize(
+    "model_w_task, injection_policy",
+    [
+        (("google/t5-v1_1-small",
+          "text2text-generation"),
+         {
+             T5Block: ('SelfAttention.o',
+                       'EncDecAttention.o',
+                       'DenseReluDense.wo')
+         }),
+        (("roberta-large",
+          "fill-mask"),
+         {
+             RobertaLayer: ('output.dense')
+         }),
+    ],
+    ids=["t5",
+         "roberta"],
+)
+@pytest.mark.parametrize("world_size", [1, 2])
```

Comment:
I'm surprised this works for our `DistributedTest` class, as it was certainly not designed to work this way. The `world_size` parameter should be a property of the class. See https://github.com/microsoft/DeepSpeed/blob/799120e7e44cefde7db8005295d40c2fa1eaf28d/tests/unit/common.py#L82

Reply:
Updated to no longer parameterize the world_size.